src/utils/index.ts

```diff
@@ -11,4 +14,20 @@ const isPlatformAndroid = (): boolean => {
   return Platform.OS === "android"
 }
 
-export { DateTimeUtils, StorageUtils, isPlatformiOS, isPlatformAndroid }
+const displaySelfAssessment = (): boolean => {
```

Comment:
What do you think about inlining these to the files inwhich they are used? They arent really utilities and it would be easier to understand the business logic if it was colocated.

Reply:
I am okay with that, but those files are going to be used in a few places, are we ok repeating the same constant a few times?

Reply:
yes totally.

Reply:
well the constant is defined in the env file. so its fine to repeat the `env.CONFIG_VAR` a few times, if it improves readability and maintainability.